content/docs/views.md

```diff
@@ -0,0 +1,102 @@
+
+## Zoom
+
+The **Zoom** sub-menu will allow you to **Zoom In** (enlarge the text), **Zoom Out** (shrink the text), or **Restore Default Zoom** (which will change the text size back to default settings).  These actions all act only on the current View, so if you have two Views visible, only one will be affected.
+
+## Move / Clone
+
+The **Move/Clone Current Document** submenu will allow you to affect where the current document is viewed.
+
+If you **Move to Other View**, the current file will be moved from one View to the other.  If there is currently only one View visible, this command will make the second View visible -- it may be to the right, below, to the left, or above the original View, depending on where the second view was last placed.  To change the arrangement of the Views, right-click on the dotted bar between the two Views, and **Rotate Right** or **Rotate Left**, which will change the orientation from side-to-side to above-and-below.  You can also accomplish this **Move to Other View** by dragging the tab from one View's tab-bar to the other, or right clicking on the tab and selecting the **Move to Other View** from that menu.
+
+If you **Clone to Other View**, the current file will be visible in both Views at the same time.
+
+If the current file is currently not modified (so no unsaved changes), you can also either **Move to New Instance** or **Clone to New Instance**, which will move the active file from this instance to a new instance of Notepad++, or make this file open in both the current Notepad++ and a new instance of Notepad++.  This will work to create a new instance of Notepad++ even if the [**Settings > Preferences > Multi-Instance**](../preferences/#multi-instance) is set to "mono-instance".
+
+## Tab
+
+The **Tab** sub-menu will allow you to navigate through the various open tabs of the active View.
+
+The numbered entries will activate the _N_th tab. The **Next tab** and **Previous tab** actions will activate the next or previous tab in the list of tabs (wrapping).
+
+The **Move Tab Forward** and **Move Tab Backward** will change the current file's position in the tab list one slot at a time, forward or backward.  The tab order can also be changed by dragging the tabs to the new position in the tab-bar.
+
+## Wrapping
+
+The **Word wrap** entry will toggle whether or not long lines will be wrapped in the display.  When not set, you will have to use the horizontal scroll bar to the see the whole current line; when set, the line will _appear_ to wrap at the end of your current editor view width, but there will be no newline (so if you open the same text file in a different editor, or a different computer's Notepad++ with different settings, it won't wrap at that character).  It is called "word wrap" instead of just "line wrap", because the line wraps on whole-word increments when possible (but if the current word is longer than the entire line-length, it _will_ wrap in the middle of the word).  **_[Remembered]_**

```

Comment:
Is this **[Remembered]** needed here?

Reply:
It is used in several places below. Seems a bit unrelated.

Reply:
The word-wrap state is remembered from one run of Notepad++ to the next, which is what **_[Remembered]_** is indicating...

Reply:
... but I decided it could be made clearer that the **_[Remembered]_** applies to the toggle portion of the word-wrap; I split it into two paragraphs, with the **_[Remembered]_** at the end of the first brief description (as was used on most of the other instances of **_[Remembered]_**).  The second paragraph gives more details on the feature.

Reply:
It was described at the beginning of the document, but I agree it would be less obvious for someone who was linked somewhere in the middle of the file.  Now I am using a footnote indicator on all the toggles that are remembered from run-to-run, and that footnote indicator has hover text saying "Remembered Setting" and links to an actual footnote which describes what a "Remembered Setting" is.  Hopefully, this is more clear